app/services/model_exporters/experiment_exporter.rb

```diff
@@ -3,10 +3,8 @@
 module ModelExporters
   class ExperimentExporter < ModelExporter
     def initialize(experiment_id)
-      @experiment = Experiment.find_by_id(experiment_id)
-      raise StandardError, 'Can not load experiment' unless @experiment
```

Comment:
We don't need to handle error anymore?

Reply:
.find() throws an exception if object not found